avaframe/log2Report/generateCompareReport.py

```diff
@@ -26,7 +26,21 @@ def copyPlots(avaName, testName, outDir, plotListRep, rel):
         plotPaths[key] = plotName
 
     return plotPaths
+    
 
+def copyAimecPlots(plotFiles, testName, outDir, plotPaths):
+    """ copy the quick plots to report directory """
+
+    for pDict in plotFiles:
+        for key in pDict:
+            print('key', key, 'file', pDict[key])
```

Comment:
This is a debug print, right? -> Please remove

Reply:
done